trunk/SUAVE/Methods/Flight_Dynamics/Dynamic_Stability/compute_dynamic_flight_modes.py

```diff
@@ -126,9 +126,9 @@ def compute_dynamic_flight_modes(results,aircraft,flight_conditions,cases):
     for wing in aircraft.wings:
         if wing.control_surfaces :
             for cs in wing.control_surfaces:
-                ctrl_surf =  wing.control_surfaces[cs]
+                ctrl_surf =  cs
```

Comment:
this line can also now be removed by making the preceding line `for ctrl_surf in wing.control_surfaces`

Reply:
Addressed